### PR TITLE
Ozen

### PR DESCRIPTION
**Pull request by ozendizman**
Can you take a look

### CODE REVIEW

File: ozen.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+> This is what it is
```
Comment (kursadysr) [inline ozen.md:1]: This doesn't look good.

File: ozen.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+x : 3.9
+
+}
+
+~~~
+
+
+# This is important
+
```
Comment (kursadysr) [inline ozen.md:16]: please remove "+"

**Review by kursadysr — CHANGES_REQUESTED** [review]
need some changes

File: ozen.md (hunk 1 above)
Comment (ozendizman) [reply ozen.md:1]: You are good to give some new ideas

**Review by mstylz1985 — APPROVED** [review]
looks good

**Review by Rainman21 — APPROVED** [review]

**Pull request merged**